app/src/app/Userpages.jsx

```diff
@@ -14,7 +14,7 @@ import ErrorPage from '$shared/components/ErrorPage'
 // Userpages
 import DashboardList from '$userpages/components/DashboardPage/List'
 import CanvasList from '$userpages/components/CanvasPage/List'
-import StreamShowView from '$userpages/components/StreamPage/Show'
+import StreamShowView from '$userpages/components/StreamPage/Edit'
```

Comment:
You could rename the variable and the component names as well... also `streamShowView.pcss` while you are at it :)